internal/person/service.go

```diff
@@ -21,7 +21,7 @@ func (p *personService) GetProfile(ctx context.Context, iamID string) (Person, e
 	var personObj Person
 	person, err := p.repo.GetProfile(ctx, iamID)
 	if err != nil {
-		return Person{}, err
+		return Person{}, nil
```

Comment:
Instead of suppressing the error, I think that a better approach is to surface this error to calling function, and handle it at `person/controller.go` - catch it and return `http.StatusNotFound`
https://github.com/jobsgowhere/jobsgowhere/blob/44718e731629dfb66a208f4baebc96fe871ad676/internal/person/controller.go#L37-L39

```go
	if err != nil {
		log.Println("Error occurred personController::GetProfile" + err.Error())
		if err.Error() == "profile not found" {
			web.RespondError(ginCtx, http.StatusNotFound, ProfileNotFound, "Profile not found for the given iam_id")
		} else {
			web.RespondError(ginCtx, http.StatusInternalServerError, "internal_error", "An error occurred in the server, please retry after sometime. err="+err.Error())
		}
		return
	}
```